understanding/target-size-enhanced.html

```diff
@@ -56,144 +56,125 @@
-                        author;
+                     <dd>ターゲットのサイズがユーザエージェントによって決定され、かつコンテンツ制作者によって変更されていない。
                      </dd>
                      	                 
-                     <dt>Essential</dt> 
+                     <dt>必要不可欠</dt> 
                      
-                     <dd>A particular presentation of the target is <a href="#dfn-essential">essential</a> to the information being conveyed.
+                     <dd>そのターゲットを特定の方法で提示することが、伝達される情報において<a href="#dfn-essential">必要不可欠</a>である。
                      </dd>
                      
                   </dl>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   			
-                  <dt>Goal</dt>
-                  <dd>Controls can be operated more easily, especially on touch screens.</dd>
+                  <dt>目標</dt>
+                  <dd>特にタッチスクリーンでは、コントロールをより簡単に操作できる。</dd>
                   			
-                  <dt>What to do</dt>
-                  <dd>Make custom targets at least 44 by 44 pixels.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>カスタムターゲットを少なくとも 44×44 ピクセルにする。</dd>
                   			
-                  <dt>Why it's important</dt>
-                  <dd>Some people cannot tap small objects.</dd>
+                  <dt>なぜそれが重要か</dt>
+                  <dd>小さなオブジェクトをタップできない人々もいる。</dd>
                   		
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this success criterion is to help users who may have trouble activating
-                  a small target because of hand tremors, limited dexterity or other reasons. If the
-                  target is too small, it may be difficult to aim at the target. Mice and similar pointing
-                  devices can be hard to use for these users, and a larger target will help them greatly
-                  in having positive outcomes on the web page.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、手の震え、器用さの限界、又はその他の理由により、小さなターゲットをアクティブにすることが困難な利用者を支援することである。ターゲットが小さすぎると、ターゲットを狙うのが困難になる場合がある。マウス及び類似のポインティングデバイスは、これらの利用者にとって使用するのが困難な場合があり、より大きなターゲットは、ウェブページで良い結果を得るのに大いに役立つ。
                </p>
-               <p>Touch is particularly problematic as it is an input mechanism with coarse precision.
-                  Users lack the same level of fine control as on inputs such as a mouse or stylus.
-                  A finger is larger than a mouse pointer, and generally obstructs the user's view of
-                  the precise location on the screen that is being touched/activated.
+               <p>タッチは、精度の粗い入力メカニズムであるため、特に問題がある。利用者は、マウス、スタイラスペンなどの入力と同じレベルの細かい制御ができない。指はマウスポインタよりも大きいため、通常、タッチ又はアクティブ化されている画面上の正確な位置の利用者の視界を遮る。
```

Comment:
「画面上の正確な位置の利用者の視界を遮る。」の訳が少しわかりにくい感じがしたので (2.1 の訳ですが...)、以下のようにするのはいかがでしょうか。

```suggestion
               <p>タッチは、精度の粗い入力メカニズムであるため、特に問題がある。利用者は、マウス、スタイラスペンなどの入力と同じレベルの細かい制御ができない。指はマウスポインタよりも大きいため、通常、タッチ又はアクティブ化されている画面上の正確な位置を利用者が見ることを遮る。
```

Reply:
意訳気味に「タッチ又はアクティブ化されている画面上の正確な位置を利用者が確認できなくなる。」としてみましたがどうでしょう？

Reply:
よいと思います！